Tetrahedral_remeshing/package_info/Tetrahedral_remeshing/dependencies

```diff
@@ -1,25 +1,32 @@
-AABB_tree
 Algebraic_foundations
 Arithmetic_kernel
+BGL
 Cartesian_kernel
 Circulator
 Distance_2
 Distance_3
 Filtered_kernel
-Generator
+Hash_map
 Homogeneous_kernel
 Installation
 Interval_support
+Intersections_2
+Intersections_3
 Kernel_23
+Kernel_d
 Mesh_3
 Modular_arithmetic
 Number_types
+Polygon
 Polygon_mesh_processing
 Profiling_tools
 Property_map
+Random_numbers
+Skin_surface_3
```

Comment:
Are you sure "Tetrahedral remeshing" still depends on `Skin_surface_3`? I thought you had removed the dependency.

Reply:
I am working on it but one of my test cases is still failing (both with and without Skin_surface though, so I could/should push it soon)